packages/tgpu-cli/gen.mjs

```diff
@@ -7,56 +7,73 @@ const cwd = new URL(`file:${process.cwd()}/`);
 
 const LENGTH_VAR = 'arrayLength';
 
+let exportsList = [];
+
 /**
  * @param { string } input
  * @param { string } output
+ * @param { boolean } toTs
+ * @param { boolean } toCjs
  */
```

Comment:
Maybe it would be better to replace this with an enum?
```suggestion
/**
 * @enum {string}
 */
const Target = {
  TS: 'ts',
  CJS: 'cjs',
};

/**
 * @param { string } input
 * @param { string } output
 * @param { Target } target
 */
```

Reply:
don't think so, these flags are not mutually exclusive